frontend/src/admin/components/sidebar/dropdown/dropdown-menu.tsx

```diff
@@ -0,0 +1,35 @@
+import React, {useState} from "react";
+import MenuIcon from "./menu-icon";
+import DropdownMenuItem from "./dropdown-menu-item";
+import "./dropdown-menu.scss";
+
+export default function DropdownMenu() {
+    const [showDropdown, setShowDropdown] = useState<boolean>(false);
+
+    const toggleDropdown = () => {
+        setShowDropdown(!showDropdown);
+    };
+
+    const dismissHandler = (event: React.FocusEvent<HTMLButtonElement>): void => {
```

Comment:
`dismissHandler` refers to when the function is used. A function should be agnostic about this since we can call the function from anywhere. There is nothing stopping me from writing `<Button onClick={() => dismissHandler()}>Log 'click me' to the console</Button>`

Instead functions should be named after what they do. It looks like this hides the dropdown, so a more descriping name would be `hideDropdown()`

Reply:
✅ This is changed